approval_polls/templates/approval_polls/index.html

```diff
@@ -6,6 +6,7 @@
     {% for poll in latest_poll_list %}
       <li class='nowrap'>
         <a href='{% url 'approval_polls:detail' poll.id %}'>{{ poll.question }}</a>
+        &nbsp;&nbsp;&nbsp;&nbsp;{{ poll.user.username }}
```

Comment:
Thanks.
I just followed the format of above {{ poll.question }}
I will change it.